apriltag_ros/src/apriltag_ros_continuous_node.cpp

```diff
@@ -30,12 +30,14 @@
  */
 
 #include <ros/ros.h>
-
+#include <opencv2/opencv.hpp>
 #include <nodelet/loader.h>
 
 int main(int argc, char **argv)
 {
   ros::init(argc, argv, "apriltag_ros");
+  // Disable opencv threading optimizations
```

Comment:
Can you expand this comment to explain why we are doing this. Ie it improves performance significantly.

Reply:
Done below.